API.md

```diff
@@ -265,6 +280,13 @@ lastChangeTime: Date
 ```
 
 
+## `TrialUser`
+```
+email: String
```

Comment:
```suggestion
username: String
```

In the `/login` request, we call this `username`, even if it's usually an email. This would be more consistent.

Reply:
Similar reasoning as before; the login procedure of `symfony` expects a username; this is why it was called like this with the /login request. Here I return the `email` of the `TrialUser` (which serves "coincidentally" as the username too).

Reply:
I always felt like it was more of a coincidence that the username doubled as an email than vice versa, personally.

Reply:
Whichever way you choose, I'd want the naming to be consistent with `/login`, so either have both use `email` or both use `username`; I'd go with the latter.

Reply:
done (the latter)